car-app-api/src/test/java/com/sky/uk/carapp/service/GenerateHeaderServiceTest.java

```diff
@@ -0,0 +1,27 @@
+package com.sky.uk.carapp.service;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Objects;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class GenerateHeaderServiceTest {
+    GenerateHeaderService generateHeaderService = new GenerateHeaderService();
```

Comment:
You can use @Autowired here instead of creating a new object

Reply:
Hi @tasnimb when we used @Autowired it threw an error and stated that our object is null. We will investigate further. @metewanne